standard/expressions.md

```diff
@@ -3884,6 +3953,7 @@ The `*`, `/`, `%`, `+`, and `-` operators are called the arithmetic operators.
 ```ANTLR
 multiplicative_expression
     : range_expression
+    | switch_expression
     | multiplicative_expression '*' range_expression
     | multiplicative_expression '/' range_expression
     | multiplicative_expression '%' range_expression
```

Comment:
To add this I had to delete @BillWagner’s comment:

> This needs more work (see other comments on grammar). I added this as a placeholder to remember it while addressing merge conflicts.

I believe the intended grammar is:

```suggestion
    : switch_expression
    | multiplicative_expression '*' switch_expression
    | multiplicative_expression '/' switch_expression
    | multiplicative_expression '%' switch_expression
```